### PR TITLE
docs: fix gifs

### PR DESCRIPTION
**Pull request by azayz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Tell us what you think: <a href="https://10sw1tcpld4.typeform.com/to/VTAyYRpR?ut
 <!-- start elevator-pitch -->
 
 <p align="center">
-<img src="https://user-images.githubusercontent.com/40893766/191586156-d9e4839b-8cc6-4f68-8c13-b4e9352de21d.gif" alt="Jina NOW logo: The data structure for unstructured data" width="1225px" height="497px">
+<img src="https://user-images.githubusercontent.com/11627845/164569398-5ef22a41-e2e1-438a-88a5-2ac43ad9426d.gif" alt="Jina NOW logo: The data structure for unstructured data" width="1225px" height="497px">
```
Comment (makram93) [inline README.md:19]: It looks a bit stretched out. Can you readjust the size?
image

File: docs/main.md [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+<a href="https://pypi.org/project/jina-now/"><img src="https://github.com/jina-ai/jina/blob/master/.github/badges/python-badge.svg?raw=true" alt="Python 3.7 3.8 3.9 3.10" title="Jina NOW supports Python 3.7 and above"></a>
+<a href="https://pypi.org/project/jina-now/"><img src="https://img.shields.io/pypi/v/jina-now?color=%23099cec&amp;label=PyPI&amp;logo=pypi&amp;logoColor=white" alt="PyPI"></a>
+</p>
+
+<!-- start elevator-pitch -->
+
+<p align="center">
+
+
+````{tab} Text-to-image
```
Comment (makram93) [inline docs/main.md:21]: Looks much better now. Can we make the link to open up a new tab?

**Review by makram93 — COMMENTED** [review]
Looks much better now. Thanks for this fix.
Left some comments

**Review by joschkabraun — APPROVED** [review]
LGTM!

**Review by makram93 — APPROVED** [review]
LGTM!

**Review by florian-hoenicke — CHANGES_REQUESTED** [review]
Good way of showing the capabilities in the docs. The trade off is that we have text duplication. Can we get ride of the duplicate code?

**Pull request merged**